code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -286,3 +286,28 @@
 	name = "syringe (syriniver)"
 	desc = "Contains syriniver, used to treat toxins and purge chemicals.The tag on the syringe states 'Inject one time per minute'"
 	list_reagents = list(/datum/reagent/medicine/C2/syriniver = 15)
+
+/obj/item/reagent_containers/syringe/contraband
+	name = "unlabeled syringe"
+	desc = "A syringe containing some sort of unknown chemical cocktail."
+
+/obj/item/reagent_containers/syringe/contraband/space_drugs
+	list_reagents = list(/datum/reagent/drug/space_drugs = 15)
+
+/obj/item/reagent_containers/syringe/contraband/krokodil
+	list_reagents = list(/datum/reagent/drug/krokodil = 15)
+
+/obj/item/reagent_containers/syringe/contraband/crank
+	list_reagents = list(/datum/reagent/drug/crank = 15)
+
+/obj/item/reagent_containers/syringe/contraband/methamphetamine
+	list_reagents = list(/datum/reagent/drug/methamphetamine = 15)

```

Comment:
This is enough meth to cause a significant explosion (or two if you split it, I think) if heated, which could be used to break walls down into girders. If that's okay with you (since whoever heats the meth up will take damage from the explosion too and it's a creative way to break out of prison), that's fine, but I figured that you should be made aware of this.

Reply:
Jeez, is that how meth works in real life? Either way, I'm fine with that honestly. Emergent gameplay. If it becomes a problem, we can change it later.